src/cli.ts

```diff
@@ -125,6 +125,8 @@ sade('lighthouse-parade <url>', true)
 
     const includePathGlob: string[] = toArray(opts['include-path-glob']);
 
+    const formFactor = opts['form-factor'];
```

Comment:
Can you change the flag name to have the `lh:` prefix, similar to how we named `--lh:only-categories`? We are using that prefix for any flag that gets passed directly into lighthouse.